ramls/compositeUser.json

```diff
@@ -31,6 +31,11 @@
       "type": "object",
       "description": "Service point user",
       "$ref": "servicepointsexpandeduser.json"
+    },
+    "tokenExpiration": {
+      "type": "object",
+      "description": "The token expiration object",
+      "$ref": "tokenExpiration.json"
     }
```

Comment:
We return two properties with each token's expiration so that Stripes code can have awareness of their TTL. Since each cookie/token has an `HttpOnly` attribute, client code can't see them in order to inspect these properties.

Reply:
> We return two properties with each token's expiration so that Stripes code can have awareness of their TTL. Since each cookie/token has an `HttpOnly` attribute, client code can't see them in order to inspect these properties.

Oh, so any client cannot only use the tokens to determine whether they need to use the refresh token to get a new temporary token, they have to rely (and store) on additional state?

Reply:
@steveellis how was this comment resolved?

Reply:
Sorry @marcjohnson-kint  - I'm not sure why I resolved it. But yes, that's right clients that are consuming these `Set-Cookie` responses must rely on these additional properties in the response body to get the TTLs for the tokens. This is by design and because of the `HttpOnly` attribute. Per [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie) `HttpOnly` "forbids JavaScript from accessing the cookie" and "mitigates against XSS". If a client is respecting `HttpOnly` it shields those cookies from script.

Reply:
Thank you for responding.

Ok, I understand that. I think I've raised my concerns about this compromise (requiring separate state to understand these tokens) elsewhere and I think that feedback has been rejected so I won't repeat it here.